Adding information about new Costs lens (#20116)
DataDog/documentation: content/en/tracing/service_catalog/_index.md

Click the Settings icon on the right hand corner to hide metric columns from the service list.

### Costs view

> The **Costs tab** provides several ways to understand the costs associated with your services with [Cloud Cost Management][19]. This includes identifying potential inefficiencies, saving opportunities, and change over time. Positioning costs data alongside metrics available in Service Catalog can improve visibility into how engineering changes affect overall cloud spend. Sort the table by clicking columns to reveal services that:

Reviewer: ```suggestion
The **Costs tab** provides several ways to understand the costs associated with your services.  Powered by [Cloud Cost Management][19], you can identify potential inefficiencies, saving opportunities, and change over time. Positioning costs data alongside metrics available in Service Catalog can improve visibility into how engineering changes affect overall cloud spend. Sort the table by clicking columns to reveal services that:
```

- Have the highest amortized AWS costs in the last month

> - Show a significant change in rate of cloud spending over time 

Reviewer: ```suggestion
- Show a significant change in the rate of cloud spending over time 
```

> To access additional information about a service’s cost changes, click on the service row to open a detailed side panel. Further, click on the **View in Analytics** button to see this information on the Cloud Costs page. 

Reviewer: ```suggestion
For more information on a service's cost changes, click on the service row to open a detailed side panel. To see this information on the Cloud Costs page, click the **View in Analytics** button. 
```
